### ISSUE TITLE
New/Modified Notifications No Longer Working

### ISSUE THREAD

**Issue by jerrytdev** [ThumbWorks/sf-ios#144]

**Comment by rodericj** [comment]
Did you try on #141, @jerrytdev ?

**Comment by jerrytdev** [comment]
Yes, but it's a problem in iOS 11/12.

**Comment by rodericj** [comment]
I agree. I do think it is worth considering the lifetime of such a fix. This group is likely going to be on ios13 by mid September. There are about 8 unscheduled coffups before then. I’m down to merge it in, but I think I am unlikely to try to fix it myself :(

**Comment by jerrytdev** [comment]
There's a chance the problem is in our processing and not the scheduling.

**Comment by rodericj** [comment]
An extremely good point!!!!!
